asistente_ladm_col/logic/xtf_model_converter/iliverrors_to_errores_calidad_0_1_converter.py

```diff
@@ -1,195 +0,0 @@
-"""
-/***************************************************************************
-                              Asistente LADM-COL
-                             --------------------
-        begin           : 2021-11-24
-        git sha         : :%H$
-        copyright       : (C) 2021 by Germán Carrillo (SwissTierras Colombia)
-        email           : gcarrillo@linuxmail.org
- ***************************************************************************/
-/***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License v3.0 as          *
- *   published by the Free Software Foundation.                            *
- *                                                                         *
- ***************************************************************************/
-"""
-import uuid
-
-from qgis.PyQt.QtCore import (pyqtSignal,
-                              QObject,
-                              QCoreApplication)
-from qgis.core import (QgsVectorLayer,
-                       QgsProject,
-                       QgsVectorLayerUtils,
-                       QgsGeometry,
-                       QgsPoint)
-import processing
-
-from asistente_ladm_col.app_interface import AppInterface
-from asistente_ladm_col.config.ladm_names import LADMNames
-from asistente_ladm_col.config.quality_rule_config import (QR_ILIVALIDATORR0001,
-                                                           QRE_ILIVALIDATORR0001E01)
-from asistente_ladm_col.core.xtf_model_converter.abstract_ladm_col_model_converter import AbstractLADMColModelConverter
-from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
-from asistente_ladm_col.utils.interlis_utils import get_layer_from_xtflog
-
-
-class IliVErrorsToErroresCalidad01Converter(AbstractLADMColModelConverter):
```

Comment:
why should we delete this class? When running the quality rule it generates the next error
![Screenshot from 2022-09-19 22-17-20](https://user-images.githubusercontent.com/8827336/191159871-a8b56b17-98c1-4b59-812c-968382fa8930.png)

Reply:
Fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/3c6ab9f70fa1deaa649e4e131be1baece41343a5 I wasn't aware that the validation against the model was performed from this deleted class.